File: spec/code_samples/Php_http1/profiles/get.php
<?php

$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/profiles');
$request->setMethod(HTTP_METH_GET);

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE',
  'content-type' => 'application/json',
  'accept' => 'application/json'
));

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: spec/code_samples/Python_requests/profiles@documents/get.py
import requests

url = "https://www.directfreight.com/api/profiles/documents"

headers = {
    'accept': "application/json",
    'content-type': "application/json",
    'end-user-token': "SOME_STRING_VALUE"
    }

response = requests.request("GET", url, headers=headers)

print(response.text)